cmd/podman/machine/platform_windows.go

```diff
@@ -1,10 +1,32 @@
 package machine
 
 import (
+	"os"
+	"strings"
+
 	"github.com/containers/podman/v4/pkg/machine"
 	"github.com/containers/podman/v4/pkg/machine/wsl"
+	"github.com/sirupsen/logrus"
 )
 
-func GetSystemDefaultProvider() machine.VirtProvider {
-	return wsl.GetWSLProvider()
+func GetSystemProvider(name string) machine.VirtProvider {
+	provider := name
+	providerOverride, found := os.LookupEnv("CONTAINERS_MACHINE_PROVIDER")
+	if found {
+		provider = providerOverride
+	}
+	provider = strings.ToUpper(strings.TrimSpace(provider))
+	switch provider {
+	case "WSL":
+		return wsl.GetWSLProvider()
+	case "QEMU":
+		logrus.Info("QEMU provider is work in progress https://github.com/containers/podman/issues/13006. Will use default provider.")
```

Comment:
This line is good now, but this is the kind of line that can suffer bit rot in the not-too-distant future.

Reply:
Agreed. This was ok for a concept, but I'm ok to change this. 👍

Reply:
Removed QEMU part on Windows, will not be part of this PR. It was included only for demo purposes and now we have HyperV to play this role.